src/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -87,6 +94,10 @@ export class CaseResolver implements Resolve<CaseView> {
   private checkAuthorizationError(error: any) {
     // TODO Should be logged to remote logging infrastructure
     console.error(error);
+    if (CaseResolver.EVENT_REGEX.test(this.previousUrl) && error.status === 404) {
```

Comment:
can we move `CaseResolver.EVENT_REGEX.test(this.previousUrl) && errorStatus === 404` into a private method `cannotAccessUpdatedCase(error.status)`? So it would read:
```
if (!cannotAccessUpdatedCase(error.status)) 
```
a bit more self explanatory I think